080-Recipes/010-csv-data/010-import-csv-data-UI.mdx

```diff
@@ -0,0 +1,109 @@
+- The table name should consist of alphanumeric characters, hyphens (-), and underscores(\_). Special characters are not allowed.
+
+### Delimiter
+
+A delimiter acts as dividers and distinguish individual data fields within a CSV file. They are useful in structuring the data, allowing for accurate interpretation and parsing.
+
+- The CSV import function has the capability to automatically identify the delimiter used in your CSV file. In case your file follows an uncommon format, you have the option to adjust the delimiter using the Xata UI.
+- The CSV import functionality supports common delimiters such as commas (,), semicolons (;), quotes ( ", ' ), braces ({}), pipes (|), or slashes ( / \ ). You can modify the delimiter in the UI, when importing the CSV file data.
+
+## Import a CSV file using the UI
+
+Xata offers an import tool directly within your dashboard Workspace. This tool allows you to import exist files into the Xata database without causing any downtime. As an alternative, you also have the option to import CSV data using the CLI; however, it's worth noting that the UI method is the recommended approach.
+Importing the CSV file involves transfering data as well as establishing a schema. Once imported, the data in the CSV file will be accessible as a table in your database Workspace.
+
+1.  Log in to your Xata account and go to the desired Workspace and database. If you haven't created a Workspace as yet, click **Add a Workspace** to do so.
+2.  Choose the preferred branch for your work.
+3.  The left-hand menu under "Tables," click **Add a table** > **Import a CSV file**.
+    ![Import CSV files](../images/import-csv-file.png)
+4.  Select **Browse Desktop** to choose a supported file to import. Ensure the filename adheres to the formatting recommendations.
+5.  Click **Begin import** to initiate the import. You'll have access to partially imported records as the process completes.
+
+<Alert status="warning">
+  To ensure all data from the chosen CSV file is imported successfully, keep the tab open until the import process
+  finishes.
+</Alert>
+
+## View imported CSV table
+
+After importing CSV data into Xata, you can use sorting and filtering capabilities to enhance data organization and retrieval from your table.
+
+Xata allows you to sort your imported data based on specific columns. This can help you arrange information in ascending or descending order, making it easier to identify trends, patterns, or specific values within your dataset.
+To sort data after importing your CSV file, navigate to the relevant table and select the column header you want to sort by. Xata will arrange the data based on your choice.
+
+Filtering allows you to narrow down the data you're working with. To filter data, access the table containing your imported CSV data and select the filter options available for each column and format. You can apply filters based on conditions or values that you want to isolate.
```

Comment:
```suggestion
```

This isn't really related to CSV import at all? If we want this content I think it should live in a different page in a seperate PR.